.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main, fix-ci-for-tests.py3.8-2.6]
```

```suggestion
    branches: [main]
```
gentle reminder comment to remove this before merging